packages/design-system/src/components/web-components/ds-choice-list/ds-choice-list.stories.tsx

```diff
@@ -0,0 +1,188 @@
+      description: {
+        component:
+          'For information about how and when to use this component, refer to the [checkbox](https://design.cms.gov/components/checkbox/) and [radio](https://design.cms.gov/components/radio/) documentation pages. Checkboxes and radios can be managed as a group using `<ChoiceList>` or individually using `<Choice>`. Note that each of the items in the `choices` array represents props that will be passed to an individual `<Choice>` component. You can therefore define any of the props listed in the `<Choice>` props table below, including all valid attributes of the [HTML input element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).',
+      },
+      componentEvents: {
+        'ds-change': {
+          description:
+            'A callback function that\'s invoked when a month\'s checked state is changed. Note: This callback is not called when a month is selected or deselected via the "Select all" or "Clear all" buttons – use the `onSelectAll` and `onClearAll` event handlers for those instances.',
+        },
+        'ds-blur': {
+          description: 'Called anytime any choice is blurred.',
+        },
+        'ds-component-blur': {
+          description:
+            'Called when any choice is blurred and the focus does not land on one of the other choices inside this component (i.e., when the whole component loses focus).',
+        },
+      },
+    },
+  },
+  decorators: [webComponentDecorator],
+};
+
+const Template = (args) => {
+  useEffect(() => {
+    const onChange = (event) => {
+      action('ds-change')(event);
+    };
+    const onBlur = (event) => {
+      action('ds-blur')(event);
+    };
+    const onComponentBlur = (event) => {
+      action('ds-component-blur')(event);
+    };
+    const choiceList = document.querySelector('ds-choice-list');
+    choiceList.addEventListener('ds-change', onChange);
+    choiceList.addEventListener('ds-blur', onBlur);
+    choiceList.addEventListener('ds-component-blur', onComponentBlur);
+    return () => {
+      choiceList.removeEventListener('ds-change', onChange);
+      choiceList.removeEventListener('ds-blur', onBlur);
+      choiceList.removeEventListener('ds-component-blur', onComponentBlur);
+    };
+  });
+
+  return <ds-choice-list {...args} />;
+};
+
+export const Default = Template.bind({});
+
+const htmlChoices = (
+  <>
+    <ds-choice type="checkbox" label="Foo" value="foo" />
+    <ds-choice type="checkbox" label="Bar" value="bar">
+      <div slot="checked-children">
+        <ds-alert heading="You'll save more with this option">
+          Based on the household information you provided, this option will give you the maximum
+          savings. We are adding some filler text just to show what it looks like when you have a
+          long alert as the checkedChildren of a Choice component.
+        </ds-alert>
+      </div>
+    </ds-choice>
+    <ds-choice type="checkbox" label="Baz" value="baz">
```

Comment:
I think it would be helpful for people interacting with this story if the checkbox label or hint text explained what these checked children are doing. It's a little weird to see the unchecked children right from the get-go with the "Foo Bar Baz", even if it's useful to us for visual regression testing.

I think we should also apply the same `ds-c-choice__checkedChild` class that we do in the React ChoiceList story for consistency.

Reply:
Should we remove the unchecked markup?

Reply:
If you can come up with something that seems plausible, it seems fine to me to include it. But if you can't come up with something plausible, removing it sounds fine too.